vibi-dpu/src/core/coverage.rs

```diff
@@ -36,25 +36,25 @@ pub async fn process_coverage(hunkmap: &HunkMap, review: &Review, repo_config: &
                 add_bitbucket_reviewers(&prhunk, hunkmap, review, &access_token).await;
             }
             if review.provider().to_string() == ProviderEnum::Github.to_string() {
-                add_github_reviewers(prhunk, review, &access_token).await;
+                add_github_reviewers(review, &coverage_map, &access_token).await;
             }
         }  
     }
 }
 
-async fn add_github_reviewers(prhunk: &PrHunkItem, review: &Review, access_token: &str) {
+async fn add_github_reviewers(review: &Review, coverage_map: &HashMap<String, (String, Option<Vec<String>>)>, access_token: &str) {
     let mut reviewers: HashSet<String> = HashSet::new();
-    for blame in prhunk.blamevec() {
-        let blame_author_opt = get_blame_user(blame, review, access_token).await;
-        log::debug!("[add_github_reviewers] blame_author_opt = {:?}", &blame_author_opt);
-        if blame_author_opt.is_none() {
+    for (_, (_, provider_ids_opt)) in coverage_map.iter() {
+        if provider_ids_opt.is_none() {
             continue;
         }
-        let blame_author = blame_author_opt.expect("Empty blame_author_opt");
-        if reviewers.contains(&blame_author) || prhunk.author().to_string() == blame_author {
+        let provider_ids = provider_ids_opt.to_owned().expect("Empty provider_ids_opt");
+        let provider_id_opt = provider_ids.first();
```

Comment:
Assumption: We assumed that single email can have multiple github or bitbucket handles. So we just go for the first handle from the array of handles.